File: Task8/ex1.py
using namespace std;

int main() {
  long long n;
    
  cin >> n;
 
  do 
    {
       if (n>10) cout << n%10 << " ";
       else cout << n%10;
       n=n/10;
    }  
  while (n>0);    
}

File: 7class/ex1.cpp
using namespace std;

int main() {

  int n, i;
  cin >> n;
  for (i=1; i<=n; i++)
   {
       cout << i << " + " << n-i+1 << " = " << n+1;
       if (i<n) cout << endl;
   }
}